siege_utilities/geo/providers/redistricting_data_hub.py

```
@@ -0,0 +1,1084 @@
+    if not population_cols:
+        log_warning("No population columns found for demographic overlay")
+        return pd.DataFrame()
+
+    # Ensure same CRS
+    if plan_gdf.crs != census_gdf.crs:
+        census_gdf = census_gdf.to_crs(plan_gdf.crs)
+
+    joined = gpd.sjoin(census_gdf, plan_gdf[[district_id_col, "geometry"]], predicate="within")
+
+    result = joined.groupby(district_id_col)[population_cols].sum().reset_index()
+    return result
```

Ruff (0.15.11)

[warning] Unnecessary assignment to `result` before `return` statement

Remove unnecessary assignment

(RET504)